config/setup/templates.go

```diff
@@ -32,18 +32,48 @@ func templatesParse(c *Controller) ([]templates.Rule, error) {
 	for c.Next() {
 		var rule templates.Rule
 
-		if c.NextArg() {
+		rule.Path = defaultTemplatePath
+		rule.Extensions = defaultTemplateExtensions
+
+		args := c.RemainingArgs()
+
+		switch len(args) {
+		case 0:
+			// Optional block
+			for c.NextBlock() {
+				switch c.Val() {
+				case "path":
+					args := c.RemainingArgs()
+					if len(args) != 1 {
+						return nil, c.ArgErr()
+					}
+					rule.Path = args[0]
+
+				case "exten":
+					args := c.RemainingArgs()
+					if len(args) == 0 {
+						return nil, c.ArgErr()
+					}
+					rule.Extensions = args
+
+				case "delims":
```

Comment:
I know the method to change this in the Go std lib is `Delims()` but what if we change this word to `between`? It might make more sense to less technical users who don't know much about what a "delimiter" is.